src/assets/CollectiblesController.test.ts

```diff
@@ -1579,9 +1537,9 @@ describe('CollectiblesController', () => {
 
       expect(
         collectiblesController.findCollectibleByAddressAndTokenId(
-          collectible.address,
-          collectible.tokenId,
-        )?.transactionId,
+          mockCollectible.address,
+          mockCollectible.tokenId,
+        )?.collectible.transactionId,
```

Comment:
Do you need the `?` here?

Reply:
Yes, because the object is possibly null

Reply:
Same argument as above :)